web/app/test/ServiceMeshTest.jsx

```diff
@@ -1,6 +1,7 @@
 import _ from 'lodash';
 import { expect } from 'chai';
 import { mount } from 'enzyme';
+import multiDeployRollupFixtures from './fixtures/multiDeployRollup.json';
```

Comment:
the only deploy in here appears to be "default/hello" 
additionally, the tests pass using `metrics: []` so we should check whether we actually need these

Reply:
yep that works - i removed it